### DISCUSSION TITLE
Does doks support forestry ?

### DISCUSSION THREAD

**Discussion by vidokun** [thuliteio/doks#271] [Q&A] [answered]
sorry, let me ask, does doks already support forestry? because I want to try using doks but before that I have to make sure docs can run in forestry, thanks btw nice theme i really love it

**Comment by h-enk** [comment, marked as answer]
@vidokun, thanks! The Doks project directory structure follows /extends Hugo's. So, I expect Doks to work with Forestry. I'm planning to set up a POC + instructions. Not sure when it will be available though.

**Reply by vidokun** [reply]
Wow thanks, I tried it, and it's amazing

![image](https://user-images.githubusercontent.com/23431072/117559278-25ce3080-b0ae-11eb-8c74-02f86ab33fd8.png)
